<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddUserData</class>
 <widget class="QDialog" name="AddUserData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Додати користувача</string>
  </property>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>356</x>
     <y>128</y>
     <width>314</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Послуги</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PhoneNumberLineEdit">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>215</y>
     <width>315</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>096123456789</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>71</y>
     <width>315</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Ім'я</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>11</y>
     <width>315</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Прізвище</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PhotoPathLineEdit">
   <property name="geometry">
    <rect>
     <x>356</x>
     <y>215</y>
     <width>314</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Посилання на Google Drive</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>356</x>
     <y>11</y>
     <width>314</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Кінець виконання замовлення</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameLineEdit">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>97</y>
     <width>315</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="maxLength">
    <number>32</number>
   </property>
   <property name="placeholderText">
    <string>Ім'я</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AdressLineEdit">
   <property name="geometry">
    <rect>
     <x>356</x>
     <y>97</y>
     <width>314</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>64</number>
   </property>
   <property name="placeholderText">
    <string>Адреса</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>246</y>
     <width>315</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата оформлення замовлення</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>352</x>
     <y>279</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Ціна послуги: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="PriceLabel_2">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_13">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>грн.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDateEdit" name="StartDateEdit">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>272</y>
     <width>315</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumDate">
    <date>
     <year>2022</year>
     <month>9</month>
     <day>14</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>356</x>
     <y>71</y>
     <width>314</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Адреса</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="EndOrderDate">
   <property name="geometry">
    <rect>
     <x>356</x>
     <y>40</y>
     <width>314</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumDate">
    <date>
     <year>2022</year>
     <month>9</month>
     <day>14</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>356</x>
     <y>189</y>
     <width>314</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Фото</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddUserButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>343</y>
     <width>326</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Додати</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SurnameLineEdit">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>40</y>
     <width>315</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="maxLength">
    <number>32</number>
   </property>
   <property name="placeholderText">
    <string>Прізвище</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MiddleNameLineEdit">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>155</y>
     <width>315</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="maxLength">
    <number>32</number>
   </property>
   <property name="placeholderText">
    <string>По-батькові</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="StatusCheckBox">
   <property name="geometry">
    <rect>
     <x>354</x>
     <y>244</y>
     <width>325</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Статус замовлення</string>
   </property>
  </widget>
  <widget class="QLabel" name="ErrorLabel">
   <property name="geometry">
    <rect>
     <x>356</x>
     <y>316</y>
     <width>314</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>189</y>
     <width>315</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер телефону</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ServiceComboBox">
   <property name="geometry">
    <rect>
     <x>353</x>
     <y>151</y>
     <width>320</width>
     <height>32</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="maxVisibleItems">
    <number>12</number>
   </property>
   <property name="placeholderText">
    <string>Послуги</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>128</y>
     <width>315</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>По-батькові</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>3</width>
     <height>264</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
